flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinatorFailureTest.java

```diff
@@ -72,7 +72,7 @@ class CheckpointCoordinatorFailureTest {
 
     /**
      * Tests that a failure while storing a completed checkpoint in the completed checkpoint store
-     * will properly fail the originating pending checkpoint and clean upt the completed checkpoint.
+     * will properly fail the originating pending checkpoint and clean up the completed checkpoint.
```

Comment:
This could be seprated into a extra commit, as well as `CompletedCheckpointStoreTest`

Reply:
restore the typo. :)